src/chrome/content/rules/coin.space.xml

```diff
@@ -1,16 +1,9 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://ru.coin.space/ => https://ru.coin.space/: (6, 'Could not resolve host: ru.coin.space')
-
--->
-<ruleset name="coin.space" default_off="failed ruleset test">
+<ruleset name="coin.space">
 	<target host="coin.space" />
-	<target host="www.coin.space" />
-	<target host="db.coin.space" />
-	<target host="live.coin.space" />
-	<target host="proxy.coin.space" />
-	<target host="ru.coin.space" />
+	<target host="*.coin.space" />
+	<test url="http://www.coin.space/" />
+	<test url="http://xrp.coin.space/api/v1/" />
+	<test url="http://xlm.coin.space/api/v1/" />
```

Comment:
Cloud you add a comment with a hint that there is a subdomain for every cryptocurrency, so we are aware of this behavior and the requirement of the wildcard rule.

Reply:
Hi! @mahnunchik is member of coin.space team. You can check it here https://github.com/CoinSpace/CoinSpace/graphs/contributors